pages/app/components/theme-switcher.tsx

```diff
@@ -26,44 +26,72 @@ export default function ThemeSwitcher() {
   }
 
   return (
-    <SpaceBetween direction="horizontal" size="xs">
-      <label>
-        Theme
-        <select defaultValue={THEME}>
-          <option value={THEME}>{THEME}</option>
-        </select>
-      </label>
-      <label>
-        <input {...vrSwitchProps} />
-        Visual refresh
-      </label>
-      <label>
-        <input
-          id="mode-toggle"
-          type="checkbox"
-          checked={mode === 'dark'}
-          onChange={event => setMode(event.target.checked ? Mode.Dark : Mode.Light)}
-        />
-        Dark mode
-      </label>
-      <label>
-        <input
-          id="density-toggle"
-          type="checkbox"
-          checked={urlParams.density === 'compact'}
-          onChange={event => setUrlParams({ density: event.target.checked ? Density.Compact : Density.Comfortable })}
-        />
-        Compact mode
-      </label>
-      <label>
-        <input
-          id="disabled-motion-toggle"
-          type="checkbox"
-          checked={urlParams.motionDisabled}
-          onChange={event => setUrlParams({ motionDisabled: event.target.checked })}
-        />
-        Disable motion
-      </label>
-    </SpaceBetween>
+    <Popover
+      position="bottom"
+      size="large"
+      content={
```

Comment:
From our usage gudelines

> Always use a header when the popover can be dismissed.

We need to add a header

Reply:
Added a header!